src/main/java/com/TableTalk/Enterprise/dto/Game.java

```diff
@@ -1,14 +1,14 @@
 package com.TableTalk.Enterprise.dto;
 
 
-
 import com.google.gson.annotations.SerializedName;
 import lombok.Data;
 
 import java.util.List;
 import java.util.Map;
 
-public @Data class Game {
+public @Data
+class Game {
     @SerializedName("id")
```

Comment:
Technical suggestion: Make proper use of Lombok and @SerializedName, there may be a better way to format this.